sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/E2E.Tests/TestFramework/MonitorExporterTestEnvironment.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Monitor.OpenTelemetry.Exporter.E2E.Tests.TestFramework
+{
+    using System;
+
+    using Azure.Core.TestFramework;
+
+    public class MonitorExporterTestEnvironment : TestEnvironment
```

Comment:
Why the new TestEnvironment class? 
I'm guessing you want an isolated class to test with while getting the new test working.

When you're ready to merge, I expect we only need one class.

Reply:
Yes - I am planning to remove the old one as there are many things in there we don't need.